src/enkf/gridinfo_fv3reg.f90

```diff
@@ -42,7 +42,8 @@ module gridinfo
 !
 !$$$
 
-use mpisetup, only: nproc, mpi_integer, mpi_real4, mpi_comm_world
+use mpi
```

Comment:
All modules need to explicitly declare the components used.  Please replace:

`use mpi`

with

`use mpi, only: mpi_real4,mpi_comm_world`